examples/units/artist_demo.py

```diff
@@ -1,9 +1,9 @@
 """
 ============
-Artist tests
+Artist demo
```

Comment:
If, for unknown reasons, someone has other comments, can you please shorten the "lines" with one character? But definitely not required otherwise.